Hax/functions.php

```diff
@@ -52,6 +64,17 @@ function mozhacks_settings_field_share_posts() {
 	<?php
 }
 
+// Renders the Twitter account setting field to share via.
```

Comment:
It's a global setting, found on Settings > Reading. I'll set it to @mozhacks when I push this live.